lib/plugins/blocks.js

```diff
@@ -419,6 +423,11 @@ function inject (bot, { version, storageBuilder }) {
     }
   })
 
+  // Emit open_sign event
+  bot._client.on('open_sign_entity', (data) => {
+    bot.emit('signOpen', data)
```

Comment:
this isn't good, we should abstract over the packet

Reply:
I don't know what you mean. How should this be abstracted more?

Reply:
I have no idea what's in this packet, and neither will people updating mineflayer for future updates, can you atleast destructure the packet in the arg, then put it back into an object when being emitted so people reading this code will know what to expect here

Reply:
Ok abstracted the location away to a position argument